### DISCUSSION TITLE
Is netbox supported on s390x architecture?

### DISCUSSION THREAD

**Discussion by manuvkumar** [netbox-community/netbox-docker#1185] [Getting Help]
Is netbox supported on s390x architecture? Any link related to the same will be helpful

**Comment by tobiasge** [comment]
We don't build a container for this architecture, but you could build it yourself. The build needs the tools listed in https://github.com/netbox-community/netbox-docker/blob/release/build-functions/check-commands.sh and the buildx Docker plugin.
